app/assets/javascripts/components/dashboard/navigation/panels/new/new_pane.jsx

```diff
@@ -0,0 +1,7 @@
+import React from 'react';
+
+export default function NewPane() {
```

Comment:
This is called a stateless functional component. Use these when the component is changing or manipulating its own state.